bearer/bearer.go

```diff
@@ -288,10 +288,11 @@ func (b *Token) Sign(signer user.Signer) error {
 	return nil
 }
 
-// SignedData returns actual payload to sign.
+// SignedData returns Token data to be signed on behalf of the specified user.
 //
 // See also [Token.Sign].
-func (b *Token) SignedData() []byte {
+func (b *Token) SignedData(issuer user.ID) []byte {
```

Comment:
We have a set of `SignedData()` implementations like
```
eacl/table.go:func (t Table) SignedData() []byte {
object/fmt.go:func (o *Object) SignedData() []byte {
reputation/trust.go:func (x *GlobalTrust) SignedData() []byte {
session/container.go:func (x *Container) SignedData() []byte {
session/object.go:func (x *Object) SignedData() []byte {
```
Does the problem worth deviating from this pattern?

Reply:
lets give it to #483 then. This will decrease probability of incorrect struct preset